### ISSUE TITLE
rpc error: code = ResourceExhausted desc = grpc: received message larger than max (400000002 vs. 4194304)

### ISSUE THREAD

**Issue by yourtiger** [go-skynet/LocalAI#1150]
**LocalAI version:**

quay.io/go-skynet/local-ai:latest

**Environment, CPU architecture, OS, and Version:**

Linux localhost.localdomain 3.10.0-1160.99.1.el7.x86_64 #1 SMP Wed Sep 13 14:19:20 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

**Describe the bug**

rpc error: code = ResourceExhausted desc = grpc: received message larger than max (400000002 vs. 4194304)

I am using https://github.com/ymcui/Chinese-LLaMA-Alpaca Download the Chinese Alpaca-13B model and convert it to ggml model q8 through llama.cpp_ 0. gguf file and added llama. yaml file in the models directory folder, with the following content:
![image](https://github.com/go-skynet/LocalAI/assets/9493473/1562374a-25d6-46f7-9773-ac0ea164a57d)
I started it in DEBGU mode

Using Postman, send the post command as shown in the following figure
![image](https://github.com/go-skynet/LocalAI/assets/9493473/53652af5-417c-4c81-a49f-dc8bd711e0d2)

LocalAI的日志输出如下
![image](https://github.com/go-skynet/LocalAI/assets/9493473/4097998b-683b-4f97-b790-20bc7846dc7d)

What should I do,please

**Comment by yourtiger** [comment]
pkg/grpc/Client.go
![image](https://github.com/go-skynet/LocalAI/assets/9493473/4f03dc65-bf37-4eee-9934-fece22a84445)
pkg/grpc/Server.go
![image](https://github.com/go-skynet/LocalAI/assets/9493473/85ef5d61-d1f3-46de-83ef-673c31ff8f26)

**Comment by alexmccartneymoore** [comment]
I'm experiencing the same issue: trying to use embedding with a llama model and llama backend. I'd be keen to hear about any possible solutions.

**Comment by dr-baem** [comment]
I experienced the same issue:
https://github.com/go-skynet/LocalAI/issues/1114

**Comment by dr-baem** [comment]
yourtiger seems to have found a valid solution. Can someone propose a permanent Pull/Merge Request to fix the issue?

**Comment by sfxworks** [comment]
ran into this issue today when sending a lot if images

[img-5][img-4][img-3][img-2][img-1][img-0]What are in these images? Is there any difference between them?<|eot_id|>

11:32AM DBG Prompt (before templating): <|start_header_id|>user<|end_header_id|>

[img-5][img-4][img-3][img-2][img-1][img-0]What are in these images? Is there any difference between them?<|eot_id|>

11:32AM DBG Template found, input modified to: <|start_header_id|>user<|end_header_id|>

[img-5][img-4][img-3][img-2][img-1][img-0]What are in these images? Is there any difference between them?<|eot_id|>

<|start_header_id|>assistant<|end_header_id|>

11:32AM DBG Prompt (after templating): <|start_header_id|>user<|end_header_id|>

[img-5][img-4][img-3][img-2][img-1][img-0]What are in these images? Is there any difference between them?<|eot_id|>

<|start_header_id|>assistant<|end_header_id|>

11:32AM DBG Model already loaded in memory: llava-llama-3-8b-v1_1-int4.gguf
11:32AM DBG Model 'llava-llama-3-8b-v1_1-int4.gguf' already loaded
11:32AM ERR Server error error="rpc error: code = ResourceExhausted desc = SERVER: Received message larger than max (8237233 vs. 4194304)" ip=127.0.0.1 latency=351.209892ms method=POST status=500 url=/v1/chat/completions

**Comment by Nold360** [comment]
yep. just ran into it with a (as it seems) too big image and vllm with `Isotr0py/Phi-3.5-vision-instruct-AWQ` :(
